src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -461,12 +463,15 @@ public override void VisitParameter(IParameterSymbol symbol)
 
             if (includeType)
             {
-                AddRefKindIfRequired(symbol.RefKind);
-
-                if (symbol.IsParams && format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeParamsRefOut))
+                if (format.ParameterOptions.IncludesOption(SymbolDisplayParameterOptions.IncludeParamsRefOut))
                 {
-                    AddKeyword(SyntaxKind.ParamsKeyword);
-                    AddSpace();
+                    AddRefKindIfRequired(symbol.RefKind);
```

Comment:
>AddRefKindIfRequired(symbol.RefKind); [](start = 20, length = 37)

Is the change around this consistent with the current VB behavior? Is there a test for scenario when IncludeParamsRefOut is false for VB?